src/openvr/OpenVRInterface.h

```diff
@@ -246,30 +258,37 @@ class OpenVRInterface : public ARVRInterface {
 
 	void blit(int p_eye, RID *p_render_target, Rect2 *p_rect) {
 		// blits out our texture as is, handy for preview display of one of the eyes that is already rendered with lens distortion on an external HMD
-		ARVRInterface::Eyes eye = (ARVRInterface::Eyes)p_eye;
+		XRInterface::Eyes eye = (XRInterface::Eyes)p_eye;
 		RID *render_target = (RID *)p_render_target;
 		Rect2 screen_rect = *(Rect2 *)p_rect;
 
-		if (eye == ARVRInterface::EYE_LEFT) {
+		if (eye == XRInterface::EYE_LEFT) {
 			screen_rect.size.x /= 2.0;
-		} else if (p_eye == ARVRInterface::EYE_RIGHT) {
+		} else if (p_eye == XRInterface::EYE_RIGHT) {
 			screen_rect.size.x /= 2.0;
 			screen_rect.position.x += screen_rect.size.x;
 		}
 
-		VSG::rasterizer->set_current_render_target(RID());
-		VSG::rasterizer->blit_render_target_to_screen(*render_target, screen_rect, 0);
+		/* THIS BIT NEEDS REDONE
```

Comment:
This bit needs work, i believe this renders the active viewport to the window?

Reply:
Yup, that bit of code does the final blit to screen so you see one eye on the desktop.